jenkins-plugin/pom.xml

```diff
@@ -295,43 +265,36 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-cps</artifactId>
-      <version>2.61.3</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-durable-task-step</artifactId>
-      <version>2.26</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-basic-steps</artifactId>
-      <version>2.12</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-support</artifactId>
-      <version>${workflow-support-plugin.version}</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-support</artifactId>
-      <version>${workflow-support-plugin.version}</version>
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.17</version>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-scm-step</artifactId>
```

Comment:
Remember that an advantage of the BOM is that you need not even declare transitive dependencies.

Reply:
Yes there is probably more cleanup to do. I was also surprised by this one. Not sure why it is here but I agree that it could probably be removed.

Reply:
Without a BOM you are often forced to declare dependencies (or at least dependency management) on a lot of transitive dependencies (mainly plugins you make no direct reference to in either main or test sources) just to make sure Maven’s lousy built-in version guesses are not incompatible.

Reply:
Yes. I will verify if I can verify few useless deps. mvn dependency:analyze can guide me "a bit"